<launch>
  <include file="$(find velodyne)/launch/VelodyneParam.launch" />
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet" args="manager"/>

  <!--node pkg="nodelet" type="nodelet" name="Driver"
        args="load velodyne/Driver standalone_nodelet">
  </node-->
  <node pkg="nodelet" type="nodelet" name="Project3D"
        args="load velodyne/Project3D standalone_nodelet">
  </node>
  <node pkg="nodelet" type="nodelet" name="Points"
        args="load velodyne/Points standalone_nodelet">
  </node>
</launch>
